src/components/sequence-introduction/sequence-introduction.tsx

```diff
@@ -14,6 +15,10 @@ interface IProps {
 export const SequenceIntroduction: React.FC<IProps> = (props) => {
   const { sequence, username, onSelectActivity } = props;
   setQueryValue("sequenceActivity", "0");
+  const backgroundImage = sequence?.background_image ? sequence.background_image : undefined;
```

Comment:
Should be safe to just write `const backgroundImage = sequence?.background_image;` with the change below to the next line.